src/nginxconfig/i18n/pl/templates/domain_sections/rails.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 DigitalOcean
+
+This code is licensed under the MIT License.
+You may obtain a copy of the License at
+https://github.com/digitalocean/nginxconfig.io/blob/master/LICENSE or https://mit-license.org/
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions :
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import common from '../../common';
+const namedLocation = '';
+
+export default {
+    railsIsDisabled: `${common.rails} jest wyłączony.`,
+    railsCannotBeEnabledWithReverseProxy: `${common.rails} nie może zostać włączony dopóki włączony jest reverse proxy.`,
+    railsCannotBeEnabledWithPhp: `${common.rails} nie może zostać włączony dopóki włączony jest ${common.php}.`,
+    railsCannotBeEnabledWithPython: `${common.rails} nie może zostać włączony dopóki włączony jest ${common.python}.`,
+    enableRails: `${common.enable} ${common.rails}`,
+    namedLocation: `${namedLocation} location`,
+    provideANamedLocationToSetNamedLocation: 'Provide a named location address to set the "Named Location" configurations for your site.',
+    namedLocationExpectedToBeginWithAmpersand: 'Named location should begin with `@`.',
```

Comment:
```suggestion
    provideANamedLocationToSetNamedLocation: 'Provide a named location address to set the "Named Location" configurations for your site.', // TODO: translate
    namedLocationExpectedToBeginWithAmpersand: 'Named location should begin with `@`.', // TODO: translate
```
Ditto -- translate TODO comments